app_backend/tests/server_test.py

```diff
@@ -28,13 +28,29 @@ def test_root():
 
 
 def test_login():
-    response = client.get("/user/login")
-    assert response.status_code == 200
-    assert response.json() == {"msg": "This is the login page!"}
```

Comment:
Just checking, this deletion was intentional, yes?

Reply:
This was the old thing that was there and it got replaced with a `client.post` so this was very much intentional.